cli/src/main.rs

```diff
@@ -81,7 +90,9 @@ fn get_app_tar_gz_b64() -> String {
   let output = Command::new("rm").arg("app.tar.gz").output().unwrap();
 
   if output.status.code().unwrap() != 0 {
-    error!("Somehow could not delete temporary app.tar.gz file");
+    let err_str = format!("Somehow could not delete temporary app.tar.gz file");
+    eprintln!("{}", err_str);
```

Comment:
We were not showing this to the user before. What we log and what we show the user should not always be the same necessarily. Maybe let's show them "Unexpected error deleting the tarball generated from the git repository"?

Reply:
We were showing it until I overrode it with the logger. In terms of the messages had my doubts if make it the same or no, decide to use the same message to reduce lines.